public/src/components/switchboard.tsx

```diff
@@ -54,10 +65,15 @@ const useStyles = makeStyles(({ palette, spacing }: Theme) => ({
     paddingLeft: spacing(2),
     paddingRight: spacing(2),
     border: `1px solid ${palette.grey['300']}`,
+    width: '45%',
```

Comment:
At small breakpoints (below about 900px) this doesn’t look great anymore. It might be worth us adding a media query at whatever our closest breakpoint to that width is, to make the width 100% below that point.